src/components/Bounties/AwardChildBountyButton.tsx

```
@@ -0,0 +1,297 @@
+	const { network } = useNetworkSelector();
+	const { loginAddress, multisigAssociatedAddress, loginWallet } = useUserDetailsSelector();
+	const [gasFee, setGasFee] = useState<BN>(ZERO_BN);
+	const [beneficiary, setBeneficiary] = useState<string>('');
+	const [loadingStatus, setLoadingStatus] = useState<ILoading>({ isLoading: false, message: '' });
+	const [multisigData, setMultisigData] = useState<{ threshold: number; signatories: string[] }>({
+		signatories: [],
+		threshold: 0
+	});
+
+	const getOnChainTx = async () => {
+		if (!api || !apiReady || bountyIndex == null || isNaN(bountyIndex)) return null;
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)